finality-aleph/src/validator_network/service.rs

```diff
@@ -168,8 +171,9 @@ impl<D: Data, A: Data, ND: Dialer<A>, NL: Listener> Service<D, A, ND, NL> {
                     },
                     // pass the data to the manager
                     SendData(data, peer_id) => {
-                        if let Err(e) = self.manager.send_to(&peer_id, data) {
-                            info!(target: "validator-network", "Failed sending to {}: {}", peer_id, e);
+                        match self.manager.send_to(&peer_id, data) {
+                            Ok(_) => trace!(target: "validator-network", "Sending data to {}.", peer_id),
+                            Err(e) => info!(target: "validator-network", "Failed sending to {}: {}", peer_id, e),
```

Comment:
Just to make sure: are we sure there is no potential for spam with this log?

Reply:
Oh, it's _incredibly_ spammy if we fail a connection, good catch. I wanted to make sure that we know when we are not connected despite trying, but the manager status report should now be sufficient for that.

Downgraded to `trace`.